keras/callbacks.py

```diff
@@ -1782,7 +1782,7 @@ def __init__(self,
     elif mode == 'max':
       self.monitor_op = np.greater
     else:
-      if 'acc' in self.monitor:
+      if self.monitor.endswith('acc') or self.monitor.endswith('accuracy') or self.monitor.endswith('auc'):
```

Comment:
Make sure the line in under 80 chars, e.g.

```
if (self.monitor.endswith('acc') or
    self.monitor.endswith('accuracy') or
    self.monitor.endswith('auc')):
```

Reply:
Done. By the way , what python formatter does ```keras``` use ? ```yapf``` , ```black``` or something else ?